### DISCUSSION TITLE
SpiceDB "failed to listen on addr". What's wrong?

### DISCUSSION THREAD

**Discussion by jaycee999** [authzed/spicedb#1493] [Q&A] [answered]
OS:	macOS
Version:	11.5.2
BuildVersion:	20G95
Homebrew 3.2.14
Go - go1.17.1 darwin/amd64

I installed spicedb using brew according to the README and get a "FTL" error when I run the following:

 `spicedb serve --grpc-preshared-key "somerandomkeyhere" --grpc-no-tls`

Output
```
8:46AM INF set log level new level=info
8:46AM INF set tracing provider new provider=none
8:46AM INF using in-memory datastore
8:46AM INF dashboard running url=http://localhost:8080
8:46AM FTL failed to listen on addr for gRPC server addr=:50051
```

Not being too familiar with the underpinnings of gRPC and Go, any insight into what might be causing this would be appreciated.

**Comment by ecordell** [comment, marked as answer]
It's possible that you have something else running on port `:50051` locally already. Could you try running with a different port to see if that works? Something like:

```
spicedb serve --grpc-preshared-key "somerandomkeyhere" --grpc-no-tls --grpc-addr=:50052
```

**Comment by jaycee999** [comment]
That worked! I'd tried modifying the port before but I apparently skipped over that one.  😫 

Thank you 🙏
